pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -3,6 +3,9 @@
 , cacert
 , lib
 }:
+let
+  bazelPkg = bazel;
+in
 
 let
   bazelPkg = bazel;
```

Comment:
I think this is duplicated now

Reply:
Funny. I did hope these things wouldn't rebase cleanly :)

Fixed.